src/period/index.js

```diff
@@ -11,22 +11,25 @@ const { logPeriod } = require('../utils/debug');
 
 module.exports = (bridgeState, sender) => async (
   rsp,
-  chainInfo,
-  chainHeight
+  state,
+  chainInfo
```

Comment:
not a rename, actually added a fourth argument (`state`) and stopped using one of the existing (`chainHeight`)